task1/feedback.txt

```diff
@@ -2,13 +2,19 @@ Please specify the amount of time you needed in hours.
 Please put an X in the appropriate column below.
 
 Time required:
-Difficult:       [ ] too easy  [ ] spot-on        [ ] too hard
-Instructive:     [ ] a little  [ ] okay           [ ] a lot
-Understandable:  [ ] unclear   [ ] a bit unclear  [ ] clear
+Difficult:       [X] too easy  [ ] spot-on        [ ] too hard
+Instructive:     [X] a little  [ ] okay           [ ] a lot
+Understandable:  [ ] unclear   [ ] a bit unclear  [X] clear
 
 Comments:
 - Amount of time required too high? Did you have problems anywhere?
-- What was too hard or unclear?
 - What could we improve?
-For the love of god, when defining a 2 dimensional something use the x componend for left to right and y for up and down.
```

Comment:
I want to rant man ..

Reply:
I left the rant included in the source file itself xD